src/tree/localProject/ConfigTreeItem.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { ThemeIcon } from "vscode";
+import { AzExtTreeItem, TreeItemIconPath } from "vscode-azureextensionui";
+import { ConfigGroupTreeItem } from "./ConfigGroupTreeItem";
+
+export class ConfigTreeItem extends AzExtTreeItem {
+    public static contextValue: string = 'azureStaticConfig';
+    public contextValue: string = ConfigTreeItem.contextValue;
+    public readonly label: string;
+    public parent: ConfigGroupTreeItem;
+
+    public constructor(parent: ConfigGroupTreeItem, label: string) {
+        super(parent);
+        this.label = label;
+    }
+
+    public get iconPath(): TreeItemIconPath {
+        return new ThemeIcon('gear');
```

Comment:
I'm not super sold on the gear icon here but as usual pickins are slim 🤷‍♂️

Reply:
Hmm, maybe we use a more simple icon for this and the gear for its parent?

Something like this might work: 
![image](https://user-images.githubusercontent.com/5290572/108284132-7e3e0380-7139-11eb-8302-192b94688fc3.png)